.github/workflows/agw-docker-load-test.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: docker-ami
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      - v1.*
+
+jobs:
+  publish-docker-ami:
+    name: publish-docker-ami
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      CODE_DIR: "${{ github.workspace }}/experimental/cloudstrapper"
+      VARS_DIR: "${{ github.workspace }}/experimental/cloudstrapper/playbooks/roles/vars"
+      WORK_DIR: "${{ github.workspace }}/experimental/cloudstrapper/playbooks"
+      AGW_DOCKER_AMI: "ami-0979b7420bd7f8ace"
```

Comment:
Is there a way to trigger newer builds of the base images based on files that have changed? Alternatively, we do nightly builds and rotate every 7 days or so? Should not hold this PR back, but would be good to capture this somewhere.

Reply:
yes I can make the snapshot docker ami workflow use a trigger based on file changes, what files do you have in mind?